src/testdrive/src/action/postgres/execute.rs

```diff
@@ -33,26 +33,32 @@ impl Action for ExecuteAction {
         Ok(())
     }
 
-    async fn redo(&self, _: &mut State) -> Result<(), String> {
-        let (client, conn) = tokio_postgres::connect(&self.connection, NoTls)
-            .await
-            .map_err(|e| format!("connecting to postgres: {}", e))?;
-        println!(
-            "Executing queries against PostgreSQL server at {}...",
-            self.connection
-        );
-        let conn_handle = tokio::spawn(conn);
+    async fn redo(&self, state: &mut State) -> Result<(), String> {
+        let client = if self.connection.starts_with("postgres://") {
```

Comment:
Super minor, but if this is how we distinguish a named connection from an ephemeral connection, we should prohibit naming a connection with anything that starts with `postgres://`.

Reply:
Check added.